### PR TITLE
#1026: Full Feature test coverage of Census Hub - Use Chrome browser …

### PR DESCRIPTION
**Pull request by joegasewicz**
### What

The [dp-frontend-homepage-controller](https://github.com/ONSdigital/dp-frontend-homepage-controller "‌") is missing component tests, these need to be added before changing the page or data handled in the controller for Census Hub.

### How to review

pull down the code and run the following:
```
make test-component
```

### Who can review
anyone

Describe who worked on the changes, so that other people can review.
@joegasewicz & @geoffmrb

### CODE REVIEW

**Review by geoffmrb — APPROVED** [review]

**Review by nshumoogum — COMMENTED** [review]
Missing ci files for running component tests on CI/CD pipelines, also spotted a failing component test. Please run again locally again and fix thanks

**Review by nshumoogum — CHANGES_REQUESTED** [review]
See comments above

File: features/home_page_component.go [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+					Error:            "",
+					ImportStarted:    nil,
+					ImportCompleted:  nil,
+					PublishStarted:   nil,
+					PublishCompleted: nil,
+				}, nil
+			},
+		},
+		Renderer: rendererClientMock,
+		Topic: &top.ClienterMock{
```
[the remark this answers is not in the document]
Comment (joegasewicz) [reply features/home_page_component.go:87]: This has been rectified

File: features/home_page_component.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+					},
+					Description: zebedee.TimeseriesDescription{
+						CDID: "LOLZ",
+						Unit: "%",
+					},
+					URI: "test/uri/timeseries/456",
+				}, nil
+			},
+		},
+		ImageAPI: &homepage.ImageClientMock{
```
[the remark this answers is not in the document]
Comment (joegasewicz) [reply features/home_page_component.go:82]: This has been rectified

**Pull request merged**